aziotctl/src/system.rs

```diff
@@ -0,0 +1,55 @@
+use std::ffi::{OsStr, OsString};
+
+use anyhow::Result;
+use structopt::StructOpt;
+
+use aziotctl_common::{
+    get_status, get_system_logs, restart, set_log_level, Level, SERVICE_DEFINITIONS,
+};
+
+#[derive(StructOpt)]
+pub enum SystemOptions {
+    Restart(RestartOptions),
+    Status(StatusOptions),
+    Logs(LogsOptions),
+    SetLogLevel(LogLevelOptions),
+}
+
+#[derive(StructOpt)]
+#[structopt(about = "Restart the services")]
+pub struct RestartOptions {}
+
+#[derive(StructOpt)]
+#[structopt(about = "Report the status of system")]
+pub struct StatusOptions {}
+
+#[derive(StructOpt)]
+#[structopt(about = "Get logs for the services")]
+pub struct LogsOptions {
+    /// Extra args to be passed to journalctl
+    #[structopt(last = true)]
+    args: Vec<OsString>,
+}
+
+#[derive(StructOpt)]
+#[structopt(about = "Set the log level of the services")]
+pub struct LogLevelOptions {
+    #[structopt(value_name = r#"One of "trace", "debug", "info", "warn", or "error""#)]
+    log_level: Level,
```

Comment:
just use `log::Level` directly

Reply:
log isn't in this crate's toml, its getting Level from common's re-export

Reply:
If you're going to re-export it from `aziotctl-common`, continue calling it `LogLevel`.

What Daniel is saying is to not re-export it at all and instead have `aziotctl` use `log::Level` via its own `log` dependency.

Reply:
Oh, I'm surprised we don't already use `log` into `aziotctl` (though now that I think about it, why would we?). In any case, `log` is already part of `aziotctl`'s dependency graph, so you might as well add it to its `Cargo.toml` and use it directly.